README.rst

```diff
@@ -75,6 +75,34 @@ JSON and encoding JSON to BSON. See `benchmark.py`::
     10000 loops, best of 3: 0.683200120926
     bsonjs is 11.72x faster than json_util
 
+Limitations
+===========
+
+Top Level Arrays
+````````````````
+Because `libbson` does not distinguish between top level arrays and top
+level documents, neither does `python-bsonjs`. This means that if you give
+`dumps` or `dump` a top level array it will give you back a dictionary.
+Below are two examples of this behavior::
```

Comment:
Using the `.. code-block:: python` style (used above) instead of the implicit `::` works better because github will add python syntax highlighting .

Reply:
Done.